packages/cdktf/lib/terraform-resource.ts

```diff
@@ -317,21 +323,49 @@ export class TerraformResource
     return { to, from };
   }
 
+  private _buildMovedBlock() {
+    let movedBlock: { to: string; from: string } | undefined;
```

Comment:
I think you could simplify this method a little by using `{ to: string; from: string } | undefined` as a return type, removing this variable and instead directly return the object wherever you current assign to this variable 🤓

Reply:
Good point I like that much better– included 👍